src/payment/strategies/paypal-commerce/paypal-commerce-payment-processor.ts

```diff
@@ -0,0 +1,67 @@
+import { MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+
+import { OverlayPaypal } from './index';
+
+const widthWindow = 450;
+const heightWindow = 600;
```

Comment:
🍺 

- `widthWindow` => `modalWidth` OR `windowWidth`
- `heightWindow` => `modalHeight` OR `windowHeight`

I prefer to use `modal` as a name instead of `window`.
- `window` name related to the global window object so, this name could a little bit confuse;

Reply:
I agree. Thank you